README.md

```diff
@@ -442,29 +442,82 @@ module.exports = function(app) {
   });
 };
 ```
+#### Migrating models with GeoPoint types before loopback-connector-mysql@5.x
+Previous to `loopback-connector-mysql@5.x`, MySQL connector was saving and loading GeoPoint properties from the MySQL database in reverse.
+MySQL expects values to be POINT(X, Y) or POINT(lon, lat), but the connector was doing the opposite. If you have an application with a model
+that has a GeoPoint property using previous versions of this connector, you can migrate your models using the following programmatic approach:
+**NOTE** Please back up the database tables that have your application data before performing any of the steps.
+1. Create a boot script under `server/boot/` directory with the following:
+```js
+'use strict';
+module.exports = function(app) {
+  function findAndUpdate() {
+    var teashop = app.models.teashop;
+    //find all instances of the model we'd like to migrate
+    teashop.find({}, function(err, res) {
+      res.forEach(function(item, indx, array) {
+        //what we fetch back from the db is wrong, so need to revert it here
+        var newLocation = {lng: item.location.lat, lat: item.location.lng};
+        //only update the GeoPoint property for the model
+        item.updateAttribute('location', newLocation, function(err, inst) {
+          if (err)
+            console.log('update attribute failed ', err);
+          else
+        console.log('updateAttribute successful');
+        });
+      });
+    });
+  }
 
+  findAndUpdate();
+};
+```
+For the above example, the model definition is as follows:
+```json
+{
+  "name": "teashop",
+  "base": "PersistedModel",
+  "idInjection": true,
+  "options": {
+    "validateUpsert": true
+  },
+  "properties": {
+    "name": {
+      "type": "string",
+      "default": "storename"
+    },
+    "location": {
+      "type": "geopoint"
+    }
+  },
+  "validations": [],
+  "relations": {},
+  "acls": [],
+  "methods": {}
+}
+```
 
-## Running tests
+## running tests
```

Comment:
I think we should keep the original format of capitalization. e.g. `Running tests` instead of `running tests`.  
The similar changes below don't seem to be necessary to me.